README.adoc

```diff
@@ -0,0 +1,73 @@
+= About JICMP
+
+_JICMP_ is a small library to allow the use of _IPv4_ _ICMP_ (raw) packets in Java.
+
+== Build from source
+
+.Requirements
+
+* git
+* automake
+* autoconf
+* libtool
+* Java JDK 1.5+
+
+The repository has a _git_ submodule which contains _Macros_ required to compile from source code.
+
+.Clone the respository on local disk
+[source]
+----
+git clone https://github.com/OpenNMS/jicmp.git
+----
+
+.Switch into source code repository
+[source]
+----
+cd jicmp
+----
+
+.Initialize and update the git submodule.
+[source]
+----
+git submodule init
+git submodule update
+----
+
+.Update generated configuration files with
+[source]
+----
+autoreconf -fvi
+----
+
+.Generate make files and compile JICMP
+[source]
+----
+./configure
+make
+----
+
+.Install the library on your system, root permissions may required when working as non-root user.
+[source]
+----
+sudo make install
+----
+
+== Using JICMP as non-root
+
+_Mac OS X_ supports non-root _ICMP_ through the _SOCK\_DRGAM_ interface, which _JICMP_ uses by default.
+
+_Linux_ supports this as well, but you additionally need to set a sysctl _OID_ to allow ping for non-root users.
+
+You can set this temporarily by running: 
+ 
+[source]
+----
+sysctl -w net.ipv4.ping_group_range="0 429496729"
```

Comment:
The link to ip-sysctl.txt does not contain the description for `ping_group_range`. All I've found was a patch like this: https://lkml.org/lkml/2011/5/18/305. Any ideas why this is not in the latest docs?